src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -49,7 +52,9 @@ const SignInPageContent = (props) => (
             <View style={[styles.flexGrow2, styles.mb8]}>
                 <SignInPageForm style={[styles.alignSelfStretch]}>
                     <View style={[props.isSmallScreenWidth ? styles.mb8 : styles.mb15, props.isSmallScreenWidth ? styles.alignItemsCenter : styles.alignSelfStart]}>
-                        <ExpensifyWordmark />
+                        <ExpensifyWordmark
+                            containerStyles={[(props.isSmallScreenWidth && (props.environment === CONST.ENVIRONMENT.DEV || props.environment === CONST.ENVIRONMENT.STAGING)) ? styles.ml3 : {}]}
+                        />
```

Comment:
@parasharrajat How is this going to look like with the ad hoc builds? They have a special pill

Reply:
once that's done we can slap the 'ready to build' label on this and test right?

Reply:
or actually we can also spin up a new issue to handle that, this would be great to get done for ec3.